config/install/sharethis.settings.yml

```diff
@@ -1,4 +1,3 @@
-article_options: ''
```

Comment:
No we are not using hardcoded schema, these have been achieved previosly by using 'schema' sequence. Doing the clean up
Figured out this. @naveenvalecha